packages/store/src/workspace/page.ts

```diff
@@ -203,6 +203,10 @@ export class Page extends Space<FlatBlockMap> {
     );
   }
 
+  hasFrameBlock() {
```

Comment:
We should avoid adding page level API that is hard-coded to some specific block types.

Reply:
ok, so I move this method to selection manager and turn it to private